manifests/repo.pp

```diff
@@ -46,8 +46,8 @@
 
         $mongover = split($version, '[.]')
         $location = $::operatingsystem ? {
-          'Debian' => "https://${repo_domain}/apt/debian",
-          'Ubuntu' => "https://${repo_domain}/apt/ubuntu",
```

Comment:
@thaiphv Thanks for reporting this and the PR. This looks like a workaround instead of a bug fix. I’d like to investigate the root cause before making this change.

Reply:
@alexjfisher I agree that I was more of a workaround rather than a fix. I'll try out your idea and report it back.